great_tables/_utils_color.py

```diff
@@ -0,0 +1,314 @@
+# Utility functions for color handling
```

Comment:
I moved this out of the `data_color()` sub dir because (and this may be not the best reason) at least one of the related functions was being used in `_scss.py`. We can totally move this all back if you're thinking that's not worth it right now.